releasenotes/notes/cws-add-macro-agent-version-support-317539fb49fa7584.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    CWS now supports for agent version constraints for macro.
```

```suggestion
    Cloud Workload Security now has Agent version constraints for macros.
```
I think? Does a user need more explanation to understand what this means? I don't really understand just from this description.

---

Maybe : Cloud Workload Security now has Agent version constraints for Macros in SECL expressions. ?